packages/app-mobile/components/screens/ConfigScreen/SettingTextInput.tsx

```
@@ -35,10 +37,12 @@ const SettingTextInput: FunctionComponent<Props> = props => {
 				<TextInput
 					autoCorrect={false}
```

Biome (2.4.4)

[error] This JSX property is assigned multiple times.

(lint/suspicious/noDuplicateJsxProps)